packages/conf-rust-1-53/conf-rust-1-53.0.1/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "david@o1labs.org"
+authors: "David Wong"
+homepage: "https://github.com/mimoo/conf-rust-1-53"
+bug-reports: "https://github.com/mimoo/conf-rust-1-53/issues"
+dev-repo: "git+https://github.com/mimoo/conf-rust-1-53.git"
+license: "MIT"
+build: [
+  ["cargo" "--version"]
+  ["cargo" "+1.53.0" "build"]
+]
+depends: [
+  "conf-rust-2018"
+]
+synopsis: "Virtual package relying on cargo (rust build system)"
+description:
+  "This package can only install if cargo (edition=2018) is installed on the system with rust 1.53.0-x86_64-apple-darwin."
+flags: conf
+url {
+  src:
+    "https://github.com/mimoo/conf-rust-1-53/releases/download/0.1/conf-rust-1-53-0.1.tbz"
+  checksum: [
+    "sha256=0301a76554739cf82486a76c6ba15ddd397284f97e4376aef4c74c2a698e8350"
+    "sha512=d1e68341e03f559d7688a068c7ea93b64641896a8569594bd73f79dcf6c5a504ecce4e6db22054cdd7177e72be58567095ae4a043133cb4702e2e93fd9e6e801"
+  ]
+}
+x-commit-hash: "f003328b66ecf965d88502b6af15c27a35f14aa9"
```

Comment:
I’m quite confused as to why your meta package has an archive associated with it.

Reply:
It contains the dummy sources for cargo to build.

My question was more related to how "fine" should conf-rust packaging be, I thought we had come to the middle ground to package a generic "rust" and the editions

Reply:
Found it, I think nothing was really decided: https://github.com/ocaml/opam-repository/pull/13351

Reply:
yeah, idk, adding conf-rust-2018 was probably a mistake in hinsight